blobstore/src/test/java/org/jclouds/blobstore/integration/internal/BaseContainerIntegrationTest.java

```diff
@@ -584,12 +584,12 @@ public void testListMarkerPrefix() throws Exception {
       final String container = getContainerName();
       try {
          blobStore.createContainerInLocation(null, container);
-         blobStore.putBlob(container, blobStore.blobBuilder("a/a").payload("").build());
-         blobStore.putBlob(container, blobStore.blobBuilder("b/b").payload("").build());
-         ListContainerOptions options = new ListContainerOptions().afterMarker("b/").recursive();
+         blobStore.putBlob(container, blobStore.blobBuilder("a" + File.separator + "a").payload("").build());
```

Comment:
This is backwards -- the Filesystem blobstore should map the path separator and not the tests.